var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -127,11 +157,8 @@ def configure_args(self):
         else:
             config_args.append('--disable-params-check')
 
-        # Activate SIMD based on properties of the target
-        if 'avx' in self.spec.target:
-            config_args.append('--with-avx')
-        else:
-            config_args.append('--without-avx')
+        rdmac_prefix = lambda x: self.spec['rdma-core'].prefix \
+            if 'rdma-core' in self.spec else None
 
         config_args.extend(self.enable_or_disable('optimizations'))
```

Comment:
did you mean to use the enable-compiler-opt configure option here?  --enable-optimizations is different